api/types/helmchartargs.go

```diff
@@ -68,6 +68,10 @@ type HelmChart struct {
 	// Legal values: 'merge', 'override', 'replace'.
 	// Defaults to 'override'.
 	ValuesMerge string `json:"valuesMerge,omitempty" yaml:"valuesMerge,omitempty"`
+
+	// InstallCRDs specifies if Helm should also generate CustomResourceDefinitions.
```

Comment:
The comment does not match the field name.
```suggestion
	// IncludeCRDs specifies if Helm should also generate CustomResourceDefinitions.
```